dynacred/src/Storage.ts

```diff
@@ -21,7 +21,7 @@ export class StorageDB {
 
     static async get(key: string): Promise<string> {
```

Comment:
See `getObject` comment.

```suggestion
     static async get(key: string): Promise<string | undefined> {
```